### ISSUE TITLE
Cryptic error printout from Micrometer

### ISSUE THREAD

**Issue by danielkec** [helidon-io/helidon#8394]
[//]: # "Please remove these comments"
[//]: # "Provide a general summary of the issue in the Title above"

## Environment Details
* Helidon Version: 4.0.5
* Helidon SE
* JDK version: OpenJDK 21.0.1+12-29
* OS: Ubuntu 23.10 6.5.0-17-generic

----------

## Problem Description
```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running me.test.MainTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.48 s - in me.test.MainTest
[INFO] Running me.daniel.se.quickstart.DummyTest
2024.02.17 13:50:41.254 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeGauge@8c71f846
2024.02.17 13:50:41.262 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeGauge@763f8121
2024.02.17 13:50:41.262 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeGauge@8cb1f1f9
2024.02.17 13:50:41.262 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeGauge@c923efe7
2024.02.17 13:50:41.263 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeGauge@9ef2fc51
2024.02.17 13:50:41.263 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeGauge@c47c45b3
2024.02.17 13:50:41.263 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeGauge@5007121b
2024.02.17 13:50:41.264 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeGauge@d3477c9d
2024.02.17 13:50:41.264 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeFunctionCounter@f0493908
2024.02.17 13:50:41.264 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeFunctionCounter@6b6e6c61
2024.02.17 13:50:41.264 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeGauge@59417b18
2024.02.17 13:50:41.265 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeGauge@fe8717a
2024.02.17 13:50:41.265 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeFunctionCounter@57613ee1
2024.02.17 13:50:41.265 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeFunctionCounter@cbd5f394
2024.02.17 13:50:41.265 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeFunctionCounter@6df5c008
2024.02.17 13:50:41.266 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeFunctionCounter@e26a74bb
2024.02.17 13:50:41.266 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeFunctionCounter@91b6f2f0
2024.02.17 13:50:41.266 Unexpected discovery of unknown previously-created meter; creating wrapper for io.micrometer.core.instrument.composite.CompositeFunctionCounter@62ba7a3
```

## Steps to reproduce
```bash
helidon init --batch --version 4.0.5 -Dpackage=me.test  -Dflavor=se -Dapp-type=quickstart && \ 
cd ./quickstart-se && \
cat <<EOT >> ./src/test/java/me/test/DummyTest.java
package me.daniel.se.quickstart;
import io.helidon.webserver.testing.junit5.RoutingTest;
import org.junit.jupiter.api.Test;
@RoutingTest class DummyTest { @Test void test() {}}
EOT
```
And then:
```bash
mvn test
```

**Comment by tjquinno** [comment]
Looks like a duplicate of #8382 which, at this writing, has a PR awaiting review.

**Comment by tjquinno** [comment]
I was able to reproduce this and then, with the newly-merged PR for #8382 in place, I do not see the warnings.

Closing as duplicate.
